.github/workflows/check_external_links.yml

```diff
@@ -30,4 +30,4 @@ jobs:
           python -m pip install .
 
       - name: Check Sphinx external links
-        run: sphinx-build -b linkcheck ./docs/source ./test_build
+        run: sphinx-build -W -b linkcheck ./docs/source ./test_build
```

Comment:
One last thing (sorry) - can you rename the workflow / workflow steps / workflow file to be just "check sphinx links" or similar, since it's both external and internal now?

Reply:
yep! sorry I missed that